syntax = "proto3";

option go_package = ".;service";

message ImageRequest {
  bytes image_data = 1;
}

message ImageResponse {
  bytes image_data = 1;
  repeated Object objects = 2;
}

message Object {
  string category = 1;
  float value = 2;
}

service ImageService {
  rpc ProcessImage(ImageRequest) returns (ImageResponse);
}
